src/main/java/com/epam/wl/servlets/LoginServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.wl.servlets;
+
+import com.epam.wl.entities.User;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Optional;
+
+import static com.epam.wl.services.TestLoginService.userDAO;
+import static com.epam.wl.services.TestLoginService.signIn;
+
+@javax.servlet.annotation.WebServlet(name = "LoginServlet", urlPatterns = "/login")
+public class LoginServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        doGet(request, response);
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String email = request.getParameter("email");
+        String password = request.getParameter("password");
```

Comment:
Finals on variables that you don't change

Reply:
Fixed